homeassistant/components/venstar/binary_sensor.py

```diff
@@ -0,0 +1,53 @@
+"""Alarm sensors for the Venstar Thermostat."""
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_PROBLEM,
+    BinarySensorEntity,
+)
+
+from . import VenstarEntity
+from .const import DOMAIN
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities) -> None:
+    """Set up Vensar device binary_sensors based on a config entry."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    if coordinator.client.alerts is None:
+        return
+    sensors = [
+        VenstarBinarySensor(coordinator, config_entry, alert["name"])
+        for alert in coordinator.client.alerts
+    ]
+
+    async_add_entities(sensors, True)
```

Comment:
We don't need to update again if the coordinator is updated during integration setup before the platforms are loaded. We can remove the second argument `True`.